examples/gno.land/r/gnoland/valopers/valopers.gno

```diff
@@ -6,28 +6,30 @@ import (
 	"std"
 
 	"gno.land/p/demo/avl"
-	"gno.land/p/demo/dao"
 	"gno.land/p/demo/ufmt"
-	pVals "gno.land/p/sys/validators"
-	"gno.land/r/gov/dao/bridge"
-	validators "gno.land/r/sys/validators/v2"
+	"gno.land/r/demo/profile"
 )
 
 const (
-	errValoperExists        = "valoper already exists"
-	errValoperMissing       = "valoper does not exist"
-	errInvalidAddressUpdate = "valoper updated address exists"
-	errValoperNotCaller     = "valoper is not the caller"
+	ErrValoperExists        = "valoper already exists"
+	ErrValoperMissing       = "valoper does not exist"
+	ErrInvalidAddressUpdate = "valoper updated address exists"
+	ErrValoperNotCaller     = "valoper is not the caller"
 )
 
 // valopers keeps track of all the active validator operators
 var valopers *avl.Tree // Address -> Valoper
 
 // Valoper represents a validator operator profile
 type Valoper struct {
-	Name        string // the display name of the valoper
-	Moniker     string // the moniker of the valoper
```

Comment:
Why did you remove the `Moniker` field?

Reply:
Finally, I removed Name and to only have Moniker, like in the validator's config (so less confusing): https://github.com/gnolang/gno/pull/3830/files#diff-704edb6211212790bbca4dc586a318edea7b40092dd35083092c263bcb42f94cR41